parsers/ES.py

```diff
@@ -1,27 +1,87 @@
 #!/usr/bin/env python3
 
 from datetime import datetime, timedelta, timezone
+from json import loads
 from logging import Logger, getLogger
 from typing import Callable, Dict, List, Optional
 
-# package "ree" is used to parse data from www.ree.es
-# maintained on github by @hectorespert at https://github.com/hectorespert/ree
-from ree import (
-    Ceuta,
-    ElHierro,
-    Formentera,
-    Gomera,
-    GranCanaria,
-    IberianPeninsula,
-    Ibiza,
-    LanzaroteFuerteventura,
-    LaPalma,
-    Mallorca,
-    Melilla,
-    Menorca,
-    Response,
-    Tenerife,
-)
+from arrow import get, utcnow
+
+FUEL_MAPPING = {
+    "dem": "demand",
```

Comment:
Should not be included in here as demand is not a fuel.

I would create a new list with ignored values that includes dem and other values we want to ignore in the future.